.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "venv/bin/python2.7"
```

Comment:
Thing is, you have created a lot of files that are unnecessary, like this one. Your only changes seem to be in learning and text, but you have created around 50 more files.